museum/demo/account_move.xml

```diff
@@ -7,7 +7,6 @@
         <field name="move_type">in_invoice</field>
         <field name="company_id" ref="base.main_company"/>
         <field name="name">BILL/2025/06/0001</field>
-        <field name="checked" eval="True"/>
```

Comment:
Why? Is it a new issue? If yes, it should be in a separate commit with a reference to the standard change

Reply:
Thanks
If I read correctly the commit, we could replace it by review_state="reviewed"